gensim/models/wrappers/ldamallet.py

```diff
@@ -234,7 +234,7 @@ def show_topics(self, num_topics=10, num_words=10, log=False, formatted=True):
             if formatted:
                 topic = self.print_topic(i, topn=num_words)
             else:
-                topic = self.show_topic(i, topn=num_words)
+                topic = self.show_topic(i, num_words=num_words)
```

Comment:
(just to leave in review thread)
`show_topic`  doesn't have topn keyword argument but num_words, this fixes it